File: ooxoo-compiler/src/test/java/com/idyria/osi/ooxoo/compiler/emitter/java/ctypes/BoardValidationReportType.java

/**
 *
 */
package com.idyria.osi.ooxoo.compiler.emitter.java.ctypes;


		
			   
		
		
			   
			   
			   
			   
		
	  
		
			   
/**	
 *
				Gathers a validation report associated with validation
				results
			
 */
		
		
			   
			   
			   
			   
		
	  
public class BoardValidationReportType {
	
	
	
		
			   
		
		
			   
         
         
         
				
					    
/**	
 *
						The Time at which the Validation Report has been
						fetched
					
 */
				
			   
		
		protected com.idyria.osi.ooxoo.core.buffers.datatypes.DateTimeBuffer Timestamp = null;
		
	
			   
         
         
         
      
		
		protected OOXList<com.idyria.osi.ooxoo.core.buffers.common.ObjectBuffer> Error = null;
		
	
			   
         
         
         
      
		
		protected OOXList<com.idyria.osi.ooxoo.core.buffers.common.ObjectBuffer> EEPROMConfig = null;
		
	
			   
         
         
         
      
		
		protected com.idyria.osi.ooxoo.core.buffers.common.ObjectBuffer ValidationProcessReport = null;
		
	
		
	  

	public BoardValidationReportType() {
	
	}

}		

File: ooxoo-compiler/src/main/scala/com/idyria/osi/ooxoo/compiler/emitter/scala/ctypes/BoardValidationReportType.scala

/**
 *
 */
package com.idyria.osi.ooxoo.compiler.emitter.scala.ctypes;



/**	
 *
				Gathers a validation report associated with validation
				results
			
 */
class BoardValidationReportType {
    
    
    
/**	
 *
						The Time at which the Validation Report has been
						fetched
					
 */ 
	var  Timestamp : com.idyria.osi.ooxoo.core.buffers.datatypes.DateTimeBuffer = null
 
	var  Error : com.idyria.osi.ooxoo3.core.buffers.structural.XList[com.idyria.osi.ooxoo.core.buffers.common.ObjectBuffer] = null
 
	var  EEPROMConfig : com.idyria.osi.ooxoo3.core.buffers.structural.XList[com.idyria.osi.ooxoo.core.buffers.common.ObjectBuffer] = null
 
	var  ValidationProcessReport : com.idyria.osi.ooxoo.core.buffers.common.ObjectBuffer = null


}       
